tests/unit/utils.ts

```diff
@@ -121,9 +129,25 @@ export function generateCompletionData(
     "vpython"
   );
 
-  console.log(`Input .tst script: \n ${textDocument.getText()} \n`);
-
-  return getTstCompletionData(textDocument, completion);
+  // Coded test
+  if (optParameters?.cppTest && optParameters?.lineSoFar) {
+    console.log(`Input .cpp file: \n ${textDocument.getText()} \n`);
+    const enviroPath = getEnviroNameFromTestScript(tstFilepath);
+    if (enviroPath) {
+      return getCodedTestCompletionData(
+        optParameters.lineSoFar,
+        completion,
+        enviroPath
+      );
+    }
+
+    throw new ReferenceError("enviroPath is undefined.");
+
+    // TST
+  } else {
```

Comment:
can you make this be `else /* tst */` and remove the line above?

Reply:
Changed comment in 770736c06264d30fd8e6d29f6695849bfad10960 .